### ISSUE TITLE
[Windows] Has an overlay on the desktop when the application is minimized

### ISSUE THREAD

**Issue by ptduyit** [pichillilorenzo/flutter_inappwebview#2402]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

On the desktop screen, only the Flutter application is open. When the program is minimized or the window is moved to a different position, the webview area of the application prevents the mouse from interacting with the desktop.

### Expected Behavior

It should be possible to interact with the desktop using the mouse when the application is minimized or the window is moved to a different position.

### Steps with code example to reproduce

```dart
import 'package:flutter/material.dart';
import 'package:flutter_inappwebview/flutter_inappwebview.dart';

void main() {
  runApp(
    MaterialApp(
      home: Scaffold(
        body: InAppWebView(
          initialUrlRequest: URLRequest(
            url: WebUri("https://google.com"),
          ),
        ),
      ),
    ),
  );
}

```

### Stacktrace/Logs

none

### Flutter version

v3.24.3

### Operating System, Device-specific and/or Tool

Windows

### Plugin version

v6.1.5

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by pichillilorenzo** [comment]
Yep, the same when you move the Flutter window in some cases.
It should be fixed on next version `6.2.0-beta.2`.

**Comment by pichillilorenzo** [comment]
Released new version `6.2.0-beta.2` with the fix.
